docs/python/inference/conf.py

```diff
@@ -10,8 +10,6 @@
 import shutil  # noqa: F401
```

Comment:
If the documentation of onnxruntime and onnxruntime-traning are merged, it woud be better to move this file (and index.rst) the folder python, maybe remove one level of subfolders.

Reply:
Updated the folder structure.